packages/flutter/test/widgets/binding_cannot_schedule_frame_test.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
It would also be good to have an actual regression test for the issue, e.g. that calling `scheduleInitialSemantics` before a root widget is attached doesn't cause frames to pump.

Reply:
sounds good, let me try.